RealtimeCSG/Assets/Plugins/RealtimeCSG/Editor/Scripts/View/GUI/SceneViewInfoGUI/SceneViewInfo.GUI.cs

```diff
@@ -0,0 +1,55 @@
+/* * * * * * * * * * * * * * * * * * * * * *
+RealtimeCSG.SceneViewInfo.GUI.cs
+
+License:
+Author: Daniel Cornelius
+
+Description:
+Handles drawing information in the scene view, such as brush and model count.
+* * * * * * * * * * * * * * * * * * * * * */
+
+using UnityEditor;
+using UnityEngine;
+
+namespace RealtimeCSG
+{
+    internal sealed partial class SceneViewInfoGUI
+    {
+        public static void DrawInfoGUI( SceneView sceneView )
+        {
+            InitStyles( sceneView );
+            CSG_GUIStyleUtility.InitStyles();
+
+            try
+            {
+                Handles.BeginGUI();
+
+                if( !CSGSettings.ShowSceneInfo )
+                    return;
+
+                GUILayout.BeginArea( infoGUIRect /*, infoGUIBGStyle*/ );
+                {
+                    GUILayout.FlexibleSpace();
+
+                    // -Nuke: internally there is a root model that all other models are grouped under, so we'll subtract 1 to keep an accurate count of user created brushes.
+                    GUILayout.Label
+                    (
+                        ((InternalCSGModelManager.Models.Length - 1 <= 0) ? 0 : InternalCSGModelManager.Models.Length - 1).ToString( "Models:\t###0" ),
```

Comment:
Wouldn't it make more sense to use `Math.Max(0, InternalCSGModelManager.Models.Length - 1)` so you only have to fetch the property once? But you'd still end up with two method calls. An even faster approach would be using a local variable on the stack to store the intermediate value and only calculate once:
```C#
int length = InternalCSGModelManager.Models.Length - 1;
((length <= 0) ? 0 : length).ToString("Models:\t###0")
```